fb.c

```diff
@@ -402,7 +402,7 @@ int vcamfb_init(struct vcam_device *dev)
     info->fbops = &vcamfb_ops;
     info->par = dev;
     info->pseudo_palette = NULL;
-    info->flags = FBINFO_FLAG_DEFAULT;
+    info->flags = 0;
```

Comment:
Do you set the flags properly?

Reply:
Below is a commit of the Linux DRM subsystem which deals with the removal of the `FBINFO_FLAG_DEFAULT` macro:
https://github.com/torvalds/linux/commit/40e324e0d859d7645f9331d10986b2b96aed8e10

According to the commit above, the default value of fb_info.flags will be set properly using framebuffer_alloc(), which is zero in the current Linux version. So I assumed that fb_info.flags can be set to zero at initialization.

But for better portability, I think I should use framebuffer_alloc() to initialize the value of fb_info.flags instead of explicitlly setting it to zero. I will modify the code and commit it.

Reply:
Good. Always address the rationale in comments or git commit messages.